### PR TITLE
Issue54consolidate components

### PR DESCRIPTION
**Pull request by GurveerSohal**

### CODE REVIEW

**Review by filippociandy10 — APPROVED** [review]

**Comment by dimassudjito** [comment]
Checklist for code reviewers

General Check

- [x]  Check that the code compiles and runs
- [x]  Check that the code is producing the required output by comparing it to the acceptance criteria in the GitHub issues. (e.g., compare it with Figma sketch for frontend work)
- [x]  Check that the code has acceptable tests and that the tests are passing.
- [x]  Check that the PR is directed at the correct branch (develop Vs. main)
- [x]  Check that the PR does not have merge conflict
- [x]  Check that CI/CD tests are passing (failure is resolved in latest develop branch)

Implementation Review

- [x]  Understand implementation approach and give feedback on how we can improve the general approach
- [x]  Understand data structures being used in the implementation and provide feedback (i.e., is it optimal?)
- [x]  Understand external libraries used in the implementation and provide feedback (i.e., is it deprecated? Is there better libraries out there?)
- [x]  Check for repeated code and give feedback
- [x]  Check for performance-intensive code (e.g., heavy algorithm) and give feedback
- [x]  Check for dead code or commented code that should be deleted
- [x]  Check if variable names are sensible and descriptive

File: frontend/src/pages/About.js [hunk 1]
```diff
@@ -2,15 +2,33 @@ import React from 'react'
...
-            <Value />
-            <Member />
-        </div>
+        <Grid container>
+
+            <Grid item xs={12}>
+                <Banner 
+                    header='About Us' 
+                    body='Making technology accessible and useful for those who create communities.'
+                    buttonText='Learn more'
```
Comment (dimassudjito) [inline frontend/src/pages/About.js:15]: button should scroll to the adjacent section when click, we can maybe pass the href value for the button as a prop but this is not the only way to do the scrolling effect

File: frontend/src/components/about/Value.js [hunk 2]
```diff
@@ -1,78 +1,65 @@
 import React from 'react'
```
Comment (dimassudjito) [inline frontend/src/components/about/Value.js:1]: awesome refactoring nice

File: frontend/src/components/about/Value.js [hunk 3]
```diff
@@ -1,78 +1,65 @@
...
 
 function Value() {
     return (
         <main>
-            <Typography color="primary.dark" ml={30} fontSize={25} fontWeight="bold"> Our Values </Typography>
+            <Typography color="primary.dark" variant='h4' sx={{
+                p: 2,
+                pl: 4,
+                fontWeight: 'bold'
+            }}> Our Values </Typography>
```
Comment (dimassudjito) [inline frontend/src/components/about/Value.js:63]: we should add more bottom margin from the "Our Values" subtitle to the list of values

**Review by dimassudjito — CHANGES_REQUESTED** [review]
Looking almost ready, a couple things:

- add top and bottom margin from the "Meet the team" subtitle
- add more bottom margin from the team section (so it has more space from the footer)
- fix tests (can remove un-used tests) as ci/cd needs all tests to pass
- fix merge conflict

**Review by dimassudjito — APPROVED** [review]

**Pull request merged**